README.md

```diff
@@ -111,3 +105,13 @@ Examples of custom controllers can be found in the [example app](./example/contr
 ## Example app
 
 An example application can be found in [the ./example directory](./example). To run this, follow the instructions in the [README](./example/README.md).
+
+## Migrating to wizard v6
+
+* The code has been updated to es6 and requires a minimum of node v4
+* Additional controller middleware must be added by overriding one of the middleware setup methods (`middlewareSetup`, `middlewareChecks`, `middlewareActions`, and `middlewareLocals`) instead of by overriding the `constructor` or the `requestHandler` (see example above)
```

Comment:
Might be confusing whether you mean: in order to add addtional controller middleware you must override..., or that you need to add additional middleware in order to use the wizard.